active/TreeView/TreeViewDragItemsCompletedNewParent.md

```diff
@@ -0,0 +1,43 @@
+# Background
+
+TreeView has built-in support for drag&drop reorder but currently there is no easy way to figure out where the item is being dropped. 
+
+# Description
+
+TreeView's data usually has hierarchical structure by nature and "index" doesn't work very well in this scenario. Parent + SubIndex is a more accurate way to locate items in TreeView. 
+
+A `NewParent` property in TreeViewDragItemsCompletedEventArgs should be sufficient to represent the drop location. To get SubIndex, developers can do NewParent.Children.IndexOf().
+
+# Examples
+
+```C#
+private void TreeView_DragItemsCompleted(MUXC.TreeView sender, MUXC.TreeViewDragItemsCompletedEventArgs args)
+{
+    var parent = args.NewParent;
+}
+```
+
+# API Details
+
+```
+[webhosthidden]
+runtimeclass TreeViewDragItemsCompletedEventArgs
+{
+    ...
+
+    Object NewParent{ get; };

```

Comment:
OldParent isn't necessary?

Reply:
Jevan brought this up as well when I talked to him. Sounds like a good thing to have. Will add it.

Reply:
Hmm, I tried adding it but then I realized you can select multiple items and drag them together, they don't necessarily share the same parent. OldParentItem is probably not applicable here?

Reply:
Or should we make OldParent a dictionary of select items and their parents?